charts/datastore/README.md

```diff
@@ -43,6 +43,9 @@ The data-on-prem solution runs with one of 3 modes (datastore.serverMode):
 | datastore.token | Your organization token. This is the same token as that used by your Rookout ETL controller |
 | datastore.tokenFromSecret.name| Secret ref in which the Rookout token resides |
 | datastore.tokenFromSecret.key| Key of the secret in which the Rookout token resides |
+| datastore.dopContainerPort| Main container port, default: 8080 |
```

Comment:
it is used by both the external and the internal port, what's the reason for that? 
right now the nonroot testing open port 80 and 443 normally

Reply:
this variable only affects the internal port

Reply:
external port not controlled by any variable but hardcoded and determined by the server mode

Reply:
it seems not:
<img width="417" alt="Screen Shot 2021-01-13 at 9 28 57" src="https://user-images.githubusercontent.com/21289132/104419883-c9388a00-5581-11eb-9cda-40c141b5946e.png">

Reply:
https://github.com/Rookout/helm-charts/pull/19#discussion_r555948825